app/modules/organizations/models.py

```diff
@@ -103,26 +117,21 @@ class Organization(db.Model, HoustonModel, OrganizationEDMMixin):
     )  # pylint: disable=invalid-name
     version = db.Column(db.Integer, default=None, nullable=True)
     title = db.Column(db.String(length=50), nullable=False)
-    # todo why can dateTimes in the auth class be non Nullable but in here the upgrade fails?
-    createdDate = db.Column(
-        db.DateTime, default=datetime.datetime.now(tz=TIMEZONE), nullable=True
-    )
-    modifiedDate = db.Column(
-        db.DateTime, default=datetime.datetime.now(tz=TIMEZONE), nullable=True
-    )
 
-    logoGuid = db.Column(db.GUID, default=uuid.uuid4, nullable=True)
-    logoUrl = db.Column(db.String(length=200), nullable=True)
+    logo_guid = db.Column(db.GUID, default=uuid.uuid4, nullable=True)
+    logo_url = db.Column(db.String(length=200), nullable=True)
+
     website = db.Column(db.String(length=120), nullable=True)
 
+    members = db.relationship('OrganizationUserMemberships', back_populates='membership')
+
     def __repr__(self):
         return (
             '<{class_name}('
             'guid={self.guid}, '
             'title="{self.title}", '
             'website={self.website}, '
-            'logoUrl={self.logoUrl}, '
-            #'members={self.members}'
```

Comment:
I was going to add this back in once the sync worked. It was helpful. Why did you remove it?

Reply:
I have pushed code to re-enable `Organization.members`